decred_wallet/Features/Navigation Menu/LeftViewController.swift

```diff
@@ -72,17 +72,30 @@ class LeftViewController : UIViewController, LeftMenuProtocol {
         self.tableView.separatorColor = GlobalConstants.Colors.separaterGrey
         storyboard2 =  UIStoryboard(name: "Main", bundle: nil)
         self.tableView.registerCellClass(MenuCell.self)
-        self.totalBalance.text = ""
-        self.synIndicate.loadGif(name: "progress bar-1s-200px")
+        DispatchQueue.main.async {
+            self.totalBalance.text = ""
+            let clickGesture = UITapGestureRecognizer(target: self, action:  #selector(self.reconnect))
+            self.statusBackgroud.addGestureRecognizer(clickGesture)
+            self.scanning = UserDefaults.standard.bool(forKey: "walletScanning")
+            self.runTimer()
+            self.synIndicate.loadGif(name: "progress bar-1s-200px")
+        }
+        let initialSyncHelp = UserDefaults.standard.bool(forKey: GlobalConstants.Strings.INITIAL_SYNC_HELP)
+        if(!initialSyncHelp){
+            showAlert(message: "\nYour 33 word seed is your wallet, keep it safe. Without it your funds cannot be recovered should your device be lost or destroyed.\n\nInitial wallet sync will take longer than usual. The wallet will connect to p2p nodes to download the blockchain headers, and will fetch only the blocks that you need while preserving your privacy.", title: "Welcome to Decred Wallet.")
+            UserDefaults.standard.set(true, forKey: GlobalConstants.Strings.INITIAL_SYNC_HELP)
+            UserDefaults.standard.synchronize()
+        }
+        
         UserDefaults.standard.set(false, forKey: "synced")
         UserDefaults.standard.set(0, forKey: "peercount")
         UserDefaults.standard.synchronize()
     }
     
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(true)
-
-         print("left did open")
+        
+        // print("left did open")
```

Comment:
Delete or uncomment. I think delete, this log message isn't particularly useful.